source/features/bugs-tab.tsx

```diff
@@ -12,7 +12,7 @@ import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 import {highlightTab, unhighlightTab} from '../helpers/dom-utils';
 
-const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|(:[\w-]+:|\p{Emoji})bug)$/iu;
+const supportedLabels = /^(bug|confirmed-bug|type:bug|type\/bug|kind:bug|kind\/bug|(:[\w-]+:|\p{Emoji})bug)$/iu;
```

Comment:
We can replace the existing `:` with `[:/]`  instead

Reply:
Thanks for pointing that out. My regex knowledge is fairly limited.